config/opal_check_libnl_version.m4

```diff
@@ -0,0 +1,50 @@
+dnl -*- shell-script -*-
+dnl
+dnl Copyright (c) 2015      Research Organization for Information Science
+dnl                         and Technology (RIST). All rights reserved.
+dnl $COPYRIGHT$
+dnl
+dnl Additional copyrights may follow
+dnl
+dnl $HEADER$
+dnl
+
+dnl OPAL_CHECK_LIBNL_INIT()
+dnl --------------------------------------------------------------------
+AC_DEFUN([OPAL_CHECK_LIBNL_INIT], [
+    opal_libnl_version=0
+    opal_libnl_libs=""
+    opal_libnl3_libs=""
+])
+
+dnl OPAL_CHECK_LIBNL_VERSION(lib)
+dnl --------------------------------------------------------------------
+AC_DEFUN([OPAL_CHECK_LIBNL_VERSION], [
+     AC_LANG_PUSH(C)
+     rm -f conftest_c.$ac_ext
+      cat > conftest_c.$ac_ext << EOF
+int main(int argc, char *argv[]) {
+    return 0;
+}
+EOF
+     OPAL_LOG_COMMAND([$CC -o conftest $CFLAGS $CPPFLAGS conftest_c.$ac_ext $LDFLAGS -l$1],
```

Comment:
got that, thanks !
is there any risk we might hit the similar issue melannox folks reported ?
(libmpi had to be explicitly linked with libopen-pal e.g. linking with libopen-rte only was not enough,
and btw/fwiw, I have never been able to reproduce this)
also, that is a bit slower since linking must be performed twice to reach a conclusion
(e.g. failure to link with libnl specific symbol, and then with libnl3 specific symbol is required to conclude libXYZ do nor require libnl nor libnl3)